server/worker/main.py

```
@@ -0,0 +1,85 @@
+    pr_number = pr_data["pr_number"]
+    base_branch = pr_data["base_branch"]
+    head_branch = pr_data["head_branch"]
+
+    print(f"[Worker] Reviewing PR #{pr_number} from {pr_data['repo_name']}")
+
+    temp_dir = tempfile.mkdtemp()
+    findings = []
+
+    try:
+        # Clone repo
+        subprocess.run(["git", "clone", repo_url, temp_dir], check=True)
```

Ruff (0.13.2)

`subprocess` call: check for execution of untrusted input

(S603)